### PR TITLE
Prepare the release notes + version change for 1.1.0

### PR DESCRIPTION
**Pull request by meyskens**
Prepare the release notes + version change for 1.1

### CODE REVIEW

**Comment by meyskens** [comment]
whoops... that's a lot

**Comment by meyskens** [comment]
FIxed!

File: content/en/docs/release-notes/release-notes-1.1.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+*Note:* Let's Encrypt has announced intention to look into the possibilities of implementing this.
+
+### Error handling
+
+We improved the recognition and handling of errors given by the ACME server better. We are now able to quickly retry transient errors and surface any fatal errors faster in the Kubernetes events and logs.
+This allows you to get more insight into any rate limiting or other errors your ACME issuer provides us.
+
+## Improvements for Venafi TPP Authentication
+
+TODO: Richard
```
Comment (wallrj) [inline content/en/docs/release-notes/release-notes-1.1.md:52]:
```suggestion
It is now possible to use a long lived access-token for authentication when configuring [Venafi TPP Issuer and ClusterIssuer types](https://cert-manager.io/docs/configuration/venafi/).
This authentication mechanism is supported by `Venafi TPP >= 19.2`.
```

File: content/en/docs/release-notes/release-notes-1.1.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+All help is very appreciated and very welcome!
+
+Interested in knowing what will happen in the next releases of cert-manager? Go check out [our road map](https://github.com/jetstack/cert-manager/blob/master/ROADMAP.md)!
+
+As usual, please read the [upgrade notes](/docs/installation/upgrading/upgrading-1.0-1.1/) before upgrading.
+
+## ACME Improvements
+
+The ACME issuer is the most used cert-manager issuer. While most use is to talk to Let's Encrypt we are seeing a growing number of new ACME endpoints by certificate authorities,
```
Comment (wallrj) [inline content/en/docs/release-notes/release-notes-1.1.md:28]:
```suggestion
The ACME issuer is the most used cert-manager issuer. While most use it to talk to Let's Encrypt we are seeing a growing number of new ACME endpoints by certificate authorities,
```

File: content/en/docs/release-notes/release-notes-1.1.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+### Duration
+
+cert-manager now allows you to request certificates with a certain validity period from an ACME issuer. This allows you to get shorter or longer lived certificates from ACME solutions such as [Step-CA](https://smallstep.com/blog/private-acme-server/). You can enable this by setting `requestDuration` to `true` in the ACME Issuer configuration. Be careful, if your ACME issuer does not support this feature it is allowed by the ACME spec to hard fail the Order causing your certificate renewal or creation to stop. 
+
+*Note:* Let's Encrypt has announced intention to look into the possibilities of implementing this.
+
+### Error handling
+
+We improved the recognition and handling of errors given by the ACME server better. We are now able to quickly retry transient errors and surface any fatal errors faster in the Kubernetes events and logs.
+This allows you to get more insight into any rate limiting or other errors your ACME issuer provides us.
```
Comment (wallrj) [inline content/en/docs/release-notes/release-notes-1.1.md:48]:
```suggestion
We improved the recognition and handling of errors given by the ACME server. We are now able to quickly retry transient errors and surface any fatal errors faster in the Kubernetes events and logs.
This allows you to get more insight into any rate limiting or other errors your ACME issuer provides us.
```

**Review by wallrj — CHANGES_REQUESTED** [review]
I added a paragraph about TPP access-token credentials and spotted a couple of typos.

File: content/en/docs/installation/kubernetes.md [hunk 4]
```diff
@@ -40,10 +40,10 @@ Install the `CustomResourceDefinitions` and cert-manager itself:
 
 ```bash
 # Kubernetes 1.16+
-$ kubectl apply --validate=false -f https://github.com/jetstack/cert-manager/releases/download/v1.0.4/cert-manager.yaml
+$ kubectl apply --validate=false -f https://github.com/jetstack/cert-manager/releases/download/v1.1.0/cert-manager.yaml
```
Comment (wallrj) [inline content/en/docs/installation/kubernetes.md:43]: Is it time to remove the `--validate=false` instructions?

File: content/en/docs/installation/kubernetes.md [hunk 5]
```diff
@@ -148,10 +148,10 @@ Install the `CustomResourceDefinition` resources using `kubectl`:
 
 ```bash
 # Kubernetes 1.15+
-$ kubectl apply --validate=false -f https://github.com/jetstack/cert-manager/releases/download/v1.0.4/cert-manager.crds.yaml
+$ kubectl apply --validate=false -f https://github.com/jetstack/cert-manager/releases/download/v1.1.0/cert-manager.crds.yaml
```
Comment (wallrj) [inline content/en/docs/installation/kubernetes.md:151]: Ditto

File: content/en/docs/installation/openshift.md [hunk 6]
```diff
@@ -62,10 +62,10 @@ are included in a single YAML manifest file:
 Install the `CustomResourceDefinitions` and cert-manager itself
 ```bash
 # OpenShift 4+
-oc apply --validate=false -f https://github.com/jetstack/cert-manager/releases/download/v1.0.4/cert-manager.yaml
+oc apply --validate=false -f https://github.com/jetstack/cert-manager/releases/download/v1.1.0/cert-manager.yaml
```
Comment (wallrj) [inline content/en/docs/installation/openshift.md:65]:
```suggestion
oc apply -f https://github.com/jetstack/cert-manager/releases/download/v1.1.0/cert-manager.yaml
```

Comment (wallrj) [reply content/en/docs/installation/openshift.md:65]: Hmm, actually, I'm not sure about the `oc` tool. You know more about that than I do.

File: content/en/docs/installation/upgrading/_index.md [hunk 7]
```diff
@@ -34,10 +34,10 @@ before upgrading the Helm chart:
 
 ```bash
 # Kubernetes 1.15+
-$ kubectl apply --validate=false -f https://github.com/jetstack/cert-manager/releases/download/v1.0.4/cert-manager.crds.yaml
+$ kubectl apply --validate=false -f https://github.com/jetstack/cert-manager/releases/download/v1.1.0/cert-manager.crds.yaml
```
Comment (wallrj) [inline content/en/docs/installation/upgrading/_index.md:37]:
```suggestion
$ kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.1.0/cert-manager.crds.yaml
```

File: content/en/docs/installation/upgrading/_index.md [hunk 8]
```diff
@@ -74,8 +74,8 @@ number you want to install:
 
 ```bash
 # Kubernetes 1.15+
-$ kubectl apply --validate=false -f https://github.com/jetstack/cert-manager/releases/download/v1.0.4/cert-manager.yaml
+$ kubectl apply --validate=false -f https://github.com/jetstack/cert-manager/releases/download/v1.1.0/cert-manager.yaml
```
Comment (wallrj) [inline content/en/docs/installation/upgrading/_index.md:77]:
```suggestion
$ kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.1.0/cert-manager.yaml
```

**Review by wallrj — CHANGES_REQUESTED** [review]
[last time we did this](https://github.com/cert-manager/website/pull/309) we also updated:
 *  config/_default/params.toml
 *  scripts/gendocs/generate
 * scripts/update-content
 * scripts/verify-lint

**Comment by meyskens** [comment]
@wallrj this only adds the release notes not yet switch over the releases to be 1.1 as in the one you referred to

File: content/en/docs/installation/kubernetes.md (hunk 5 above)
Comment (meyskens) [inline content/en/docs/installation/kubernetes.md:151]:
```suggestion
$ kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.1.0/cert-manager.crds.yaml
```

File: content/en/docs/installation/kubernetes.md (hunk 4 above)
Comment (meyskens) [inline content/en/docs/installation/kubernetes.md:43]:
```suggestion
$ kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.1.0/cert-manager.yaml
```

**Review by wallrj — APPROVED** [review]
/lgtm

**Pull request merged**